imports/ui/unit/unit.jsx

```diff
@@ -278,37 +241,51 @@ class Unit extends Component {
                 >
 
                   <div className='flex-grow bg-very-light-gray'>
-                    <div className='flex pl3 pv3 bb b--very-light-gray bg-white'>
-                      <div
-                        className={'f6 fw5 ph2 ' + (showOpenCases ? 'mid-gray' : 'silver')}
-                        onClick={() => this.handleStatusClicked(true)}
-                      >
-                        Open
-                      </div>
-                      <div
-                        className={'f6 fw5 ml4 ph2 ' + (!showOpenCases ? 'mid-gray' : 'silver')}
-                        onClick={() => this.handleStatusClicked(false)}
-                      >
-                        Closed
-                      </div>
-                      <div
-                        onClick={() => this.handleAssignedClicked()}
-                        className={'f6 fw5 ml4 ph2 ' + (assignedToMe ? 'mid-gray' : 'silver')}
-                      >
-                        Assigned To Me
+                    { caseList.length ? (
+                      <div>
```

Comment:
Shouldn't this be using the same new filter and sort components as the other pages?

Reply:
the old code is replaced by filter & sorter components :)